### ISSUE TITLE
Implementation of Asymmetrical Loss

### ISSUE THREAD

**Issue by ghylander** [keras-team/keras-cv#1395]
**Short Description**

Similarly to Focal Loss, Asymmetrical Loss attempts to tackle the problem resulting from positive-negative imbalances in training data for multi-label image classification tasks.
In contrast with Focal Loss, Asymmetrical Loss treats negative and positive labels differently. From the paper's abstract:
"The loss enables to dynamically down-weights and hard-thresholds easy negative samples, while also discarding possibly mislabeled samples."

**Papers**

https://arxiv.org/abs/2009.14119

160 citations as reported by semantic scholar

**Existing Implementations**

This loss has been implemented in both PyTorch and TensorFlow. The TensorFlow implementation has been pull-requested to the tensorflow-addons repo by its author, user @dongyups (https://github.com/tensorflow/addons/pull/2748). The pull-request passes all the checks present in said repo.

The tfa team suggested the author to open a ticket here, but it seems he did not. Nor he did reply or comment on the pull request since then.

**Other Information**

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new KerasHub package, which supports image, text, and multi-modal models. Please read https://github.com/keras-team/keras-hub/issues/1831. KerasHub will support all the core functionality of KerasCV.

KerasHub can be installed with !pip install -U keras-hub. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
